sale_order_customer_no_autofollow/models/res_config_settings.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    so_no_autofollow = fields.Boolean(
+        config_parameter="so.no_autofollow", string="Customer disable autofollow"
```

Comment:
config parameter should be more explicit. Eg has a module or model name in it.
like  "sale_order.customer_no_autofollow"

Reply:
Got it, I'll fix it